src/client/components/landing/LandingBody.jsx

```diff
@@ -6,36 +6,52 @@ import Style from '../../styles/main.less';
 
 import { Canvas } from 'react-three-fiber';
 
-import { Row, Col } from 'react-bootstrap';
+import { Row, Col, Container } from 'react-bootstrap';
 
 import Scene from './Scene.jsx';
 
+import * as typeformEmbed from '@typeform/embed';
+
 import { PrimaryButton, SecondaryButton } from '../common/Buttons.jsx';
 import { WhiteLink, BlueLink } from '../common/Link.jsx';
 
 class LandingBody extends Component {
+  constructor(props) {
+    super(props);
+    this.el = null;
+  }
+  componentDidMount() {
+    if (this.el) {
+      typeformEmbed.makeWidget(this.el, "https://myriade.typeform.com/to/Y6daMA", {
+        hideFooter: true,
+        hideHeaders: true,
+        opacity: 0
+      });
```

Comment:
I was just wondering, if alternatively you can give the `div` element like an ID and call the`makeWidget` on a getElementById 🤔 inside of `componentDidMount()` instead of keeping this element using `this.el`
This is a question, what do you think? I dont have enough domain knowledge in frontend, so this is entirely your call 😄

Reply:
Yeah I chose to makewidget on the ID in componentDidMount because I wasn't sure how React deals with IDs of elements it creates in the render function